src/Protocol/RecipeMapper.cpp

```diff
@@ -0,0 +1,123 @@
+#include "Globals.h"
+#include "RecipeMapper.h"
+#include "../Root.h"
+
+cRecipeMapper::cRecipeMapper(void)
+{
+	const AString Protocols [] = {"1.12.2"};
+	for (const auto & Protocol: Protocols)
```

Comment:
How about listing the `Protocol/` folder's contents and trying each folder? That way any missing protocol can be added by the user.

Reply:
Puh, yeah, I could look into it. Tbh, I would improve it as soon as we start to add more of the configurations.
Is it desired or fine like this?